### PR TITLE
kube-ps1 0.4.0 (new formula)

### PR DESCRIPTION
**Pull request by jonmosco**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
```
==> brew audit kube-ps1 --online --new-formula
==> FAILED
Error: 1 problem in 1 formula
kube-ps1:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```
Thanks for the PR @jonmosco! We don't allow self-submissions, and this doesn't yet meet the notability requirements. You may want to host the formula in [your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap) for the time being.

**Comment by m247support** [comment]
@ilovezfs can you please confirm how @jonmosco's formula can meet notability the requirements?

**Comment by ilovezfs** [comment]
@m247support it should now.

**Comment by jonmosco** [comment]
This has > 50 stars now, but I have a new release.  Can I add to this PR?

**Comment by m247support** [comment]
@jonmosco @ilovezfs I am not a maintainer nor would I proclaim to be an expert, but  it might be worth closing this and creating a new PR for v0.3.0 as this seems to have become stale. However, I will defer to someone with a better a understanding. Thanks in advance!

**Comment by jonmosco** [comment]
Done.  Pushed update for new release.

**Comment by ilovezfs** [comment]
@jonmosco why the close?

**Comment by jonmosco** [comment]
Just thought to put in a new pull is all.  If this is still good, then awesom!

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: readme is installed automatically

Comment (jonmosco) [reply Formula/kube-ps1.rb]: Ah ok.  Should I correct this, or will it go in regardless?

Comment (ilovezfs) [reply Formula/kube-ps1.rb]: should correct

**Comment by jonmosco** [comment]
Is there anything else I need to do for this to be merged?

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: This test needs to exercise and verify the functionality of the software. See `liquidprompt` and `bash-git-prompt` for some examples.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: can we use assert_equal?

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: need to add .chomp

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: This existence test is not needed

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: comment is not needed

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: would use `share.install` instead.

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: needs to be opt_share

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: needs to be opt_share

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: needs to be opt_share

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: comment is not needed

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: needs to be opt_share

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: needs to be opt_share

File: Formula/kube-ps1.rb
Comment (ilovezfs) [inline Formula/kube-ps1.rb]: needs to be opt_share

**Comment by ilovezfs** [comment]
@jonmosco still missing a couple `opt_share`s

**Review by ilovezfs — APPROVED** [review]

**Comment by jonmosco** [comment]
Thanks for all the help.

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @jonmosco! Without people like you submitting PRs we couldn't run this project. You rock!

>Thanks for all the help.

You're welcome :)

**Pull request merged**
